cvcl/models.py

```diff
@@ -284,3 +284,58 @@ class Meta:
 
     def __str__(self):
         return str(self.id)
+
+
+class InstallPackage(models.Model):
+    name = models.CharField(max_length=30)
+    package_update = models.BooleanField(default=False)
```

Comment:
Why is package_update set for every package? It's independent of the packages you install.